### ISSUE TITLE
How about updating README to showcase this plugin

### ISSUE THREAD

**Issue by andreszs** [cubiq/ComfyUI_essentials#1]
I'm about to try this plugin since it was suggested by the IPAdapter author [here](https://civitai.com/articles/2345), and surely I'm not the only one interested in it. However, the README leaves a lot to be desired. 

There's no need to convert it to Mozilla Developer Network article either, a few screenshots to showcase the plugin provided nodes would be enough and relatively easy to add. Thanks.

**Comment by cubiq** [comment]
I know there's a note in the readme about doing better documentation. please check the included workflow, that should help a bit

**Comment by thewittyrobin** [comment]
yeah im looking it over wanting this exact kind of idea. More specifically a node that adds upscale image with model and amount. does have that idk. it doesnt tell me and idk if i wanna download this just to have to remove it.
